### PR TITLE
Storage v3

### PR DESCRIPTION
**Pull request by jenyasd209**

### CODE REVIEW

File: sdk/storage_v2_model.go [hunk 1]
```diff
@@ -225,6 +225,62 @@ type ReplicatorsPageOptions struct {
 	PaginationOrderingOptions
 }
 
+type Payment struct {
+	Replicator *Hash
```
Comment (jenyasd209) [inline sdk/storage_v2_model.go:229]: `PublicAccount`

Comment (wernling-proximax) [reply sdk/storage_v2_model.go]: Thanks =) Updated in 422f080

File: sdk/storage_v2.go [hunk 2]
```diff
@@ -96,14 +96,14 @@ func (s *StorageV2Service) GetReplicators(ctx context.Context, rpOpts *Replicato
 	return rspDTO.toStruct(s.client.NetworkType())
 }
 
-func (s *StorageV2Service) GetDownloadChannelInfo(ctx context.Context, downloadChannelId *Hash) (*DownloadChannel, error) {
+func (s *StorageV2Service) GetDownloadChannelInfo(ctx context.Context, downloadChannelId *Hash) ([]*DownloadChannel, error) {
```
Comment (jenyasd209) [inline sdk/storage_v2.go:99]: why array?

Comment (wernling-proximax) [reply sdk/storage_v2.go:99]: The function in the rest server expects an array. 
Is it possible that there can be 2 different entries having the same `DownloadChannelId`?

Comment (jenyasd209) [reply sdk/storage_v2.go:99]: It's not possible to have two same `DownloadChannelId`. So the route in REST should return `sendOne()`

Comment (wernling-proximax) [reply sdk/storage_v2.go]: I've updated in REST (https://github.com/proximax-storage/js-xpx-chain-rest/pull/131/commits/56d6fee16d364db676f253b823b82758a6d9dd23) and reverted GetDownloadChannelInfo changes (7f87330)

**Pull request merged**